cypress/e2e/shared/nav.test.ts

```diff
@@ -9,31 +9,36 @@ describe('navigation', () => {
 
   it('can navigate to each page from left nav', () => {
     // Load Data Page
-    cy.getByTestID('nav-item-load-data').click()
+    cy.getByTestID('nav-item-load-data').click({force: true})
+    cy.getByTestID('tabs').within(() => {
+      // buckets tab
+      cy.get('[id="buckets"]').click()
+      cy.url().should('contain', 'buckets')
+    })
     cy.getByTestID('load-data--header').should('exist')
 
     // Data Explorer Page
-    cy.getByTestID('nav-item-data-explorer').click()
+    cy.getByTestID('nav-item-data-explorer').click({force: true})
     cy.getByTestID('data-explorer--header').should('exist')
 
     // Dashboard Index Page
-    cy.getByTestID('nav-item-dashboards').click()
+    cy.getByTestID('nav-item-dashboards').click({force: true})
     cy.getByTestID('empty-dashboards-list').should('exist')
 
     // Tasks Index Page
-    cy.getByTestID('nav-item-tasks').click()
+    cy.getByTestID('nav-item-tasks').click({force: true})
     cy.getByTestID('tasks-page--header').should('exist')
 
     // Alerts Page
-    cy.getByTestID('nav-item-alerting').click()
+    cy.getByTestID('nav-item-alerting').click({force: true})
```

Comment:
These `force: true` clicks are necessary because new design overlays an `<a>` tag element over our buttons and this is the best way for now.

Reply:
I'm wondering if maybe we should create a Cypress command for clicking these nav items? Especially if this is something that we're going to have to do in every test that wants to click a nav bar element. Seems like something that people are going to be confused about as they are adding new tests

Reply:
Like I'm thinking a Cypress command that expands the nav bar and clicks the element (instead of doing `force: true`. The signature would be like `cy.clickNavBar('nav-item-tasks')`

Reply:
I think this is a solid idea, it'll make more sense yeah!

Reply:
I'll try to do this by seeing how similar functions were done before in the cypress setup.

Reply:
💯  awesome! yeah check the `commands.ts` file!